tests/FakeItEasy.Tests/Core/SequentialCallContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core
+{
+    using System.Collections.Generic;
+    using FakeItEasy.Core;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SequentialCallContextTests

```

Comment:
I vote for dropping this class. I don't there's any need to make `SequentialCallContext` testable in isolation.